web/src/lib/components/elements/buttons/circle-icon-button.svelte

```diff
@@ -6,7 +6,7 @@
   export let hoverColor = '#e2e7e9';
   export let padding = '3';
   export let size = '24';
-  export let title = '';
+  export let title: string;
```

Comment:
nit: Should required props be at the top of the list?

Reply:
Nice catch, fixed!